src/main/java/it/gov/pagopa/payhub/activities/dto/export/debtposition/PaidInstallmentsRequestFilterDTO.java

```diff
@@ -0,0 +1,19 @@
+package it.gov.pagopa.payhub.activities.dto.export.debtposition;
+
+
+import it.gov.pagopa.payhub.activities.dto.OffsetDateTimeIntervalFilter;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+public class PaidInstallmentsRequestFilterDTO extends OffsetDateTimeIntervalFilter {
+
+    private Long organizationId;
+    private String operatorExternalUserId;
+    private Long debtPositionTypeOrgId;
```

Comment:
don't extend the TimeInterval, just use it
```suggestion
public class PaidInstallmentsRequestFilterDTO {

    private Long organizationId;
    private String operatorExternalUserId;
    private Long debtPositionTypeOrgId;
    private OffsetDateTimeIntervalFilter paymentDate;
```